catalog/ui/catalog-ui-search/src/main/resources/OSGI-INF/metatype/org.codice.ddf.catalog.ui.config.xml

```diff
@@ -40,7 +40,7 @@
             name="Map Projection"
             description="Projection of imagery providers"
```

Comment:
I think it might be helpful to mention both projections in the description.

`Projection of imagery providers (e.g. EPSG:3857, EPSG:4326)`